### ISSUE TITLE
Not select by default seach bars

### ISSUE THREAD

**Issue by OrangeZXZ** [fast4x/RiMusic#1075]
### Description

I think that all the search bars should not be selected by default anywhere.
Maybe just on the search song page because you alrealy tap a search icon (that counts as a confirmation that you want to search)
When going in the setting you dont always want to search and it just spams my keyboard

### Suggested Solution

make the searches not selected by default (only exceptions are the search bars that pop when you press a search icon)

**Comment by OrangeZXZ** [comment]
https://github.com/fast4x/RiMusic/assets/104271365/0101d1cf-b086-45c9-805e-93cca5a81117

**Comment by Nikks84** [comment]
Yes, it is very annoying, when we want to search ourselves, the keyboard should be visible.

**Comment by dAtA-TRoN** [comment]
It should be a consensus that search bars are preselected when initiating a search by tapping on the corresponding icon. You've rightly pointed that out as well. Moreover, it's essential that in the settings, search is not preselected. Indeed, this is something that fast4x should address.

**Comment by fast4x** [comment]
i agree

**Comment by fast4x** [comment]
Improved, available from version 0.6.27

**Comment by fast4x** [comment]
For this ticket, meanwhile I have aligned the behavior for all the search bars, in future it will be improved overall.

**Comment by dAtA-TRoN** [comment]
> For this ticket, meanwhile I have aligned the behavior for all the search bars, in future it will be improved overall.

Does this mean that all search bars are now standardly deselected or selected?
